access/jira/client.go

```diff
@@ -39,6 +39,9 @@ const (
 
 	jiraMaxConns    = 100
 	jiraHTTPTimeout = 10 * time.Second
+	// Jira has a 300,000 character limit for the reason field so we
+	// truncate all reasons to a generous but conservative limit
+	jiraReasonLimit = 30000
```

Comment:
It looks like the limit might be 4096, not 300,000. When I run the test below, I get:

```
Error Trace:	jira_test.go:346
Error:      	Received unexpected error:
            	access request reason is too long, max 4096 bytes
Test:       	TestJira/TestIssueCreationWithLargeRequestReason
```

Can you add a link to any documentation in the comment which describes the limit?

Reply:
Have set the limit back to 3000 and updated the comment. 
Tried to find some documentation specifying the reason length but was unable to, beyond the info in the initial ticket.

Reply:
The 4k limit is not imposed by Jira, it's a Teleport-enforced limit.

https://github.com/gravitational/teleport/blob/368f3c31abfd74456267ba1dadd1c5e5914bbd7e/lib/services/access_request.go#L40

This was set in response to a security finding that an attacker could force Teleport to allocate large amounts of memory by submitting access requests with a huge reason field.